mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.h

```diff
@@ -3,6 +3,7 @@
 #include "vgui_controls/EditablePanel.h"
 #include "steam/isteamhttp.h"
 #include "mom_shareddefs.h"
+#include "filesystem.h"
```

Comment:
Is it possible to forward declare the things needed from this? Filesystem could be a pretty big include to have in this header

Reply:
possibly yes, I'm not sure though what the best way to forward-declare the enum is. I read you can do that with c++11 but GCC doesn't seem to like it

Reply:
Then no worries, it can be figured out in the future when the leaderboards are expanded upon in 0.11.0.